site/pages/api/create-consequence-network.api.ts

```diff
@@ -71,10 +72,26 @@ const createConsequenceNetwork = async (req: NextApiRequest, res: NextApiRespons
                 ? DISRUPTION_DETAIL_PAGE_PATH
                 : REVIEW_DISRUPTION_PAGE_PATH;
 
+        if (addAnotherConsequence) {
+            const disruption = await getDisruptionById(validatedBody.data.disruptionId, session.orgId, !!template);
+            if (!disruption) {
+                throw new Error("No disruption found to add another consequence");
+            }
+            const nextIndex = getLargestConsequenceIndex(disruption) + 1;
+            redirectToWithQueryParams(
```

Comment:
If we create a new disruption from template and in the consequence details page we click add another consequence - fill in the details and click on save button we are redirected to review disruption page but if you notice the url there two query params with the same key of `return` which has different values

Reply:
@DeannaSharma  will this be resolved in your ticket?